### PR TITLE
Resize image

### PR DESCRIPTION
**Pull request by vipasyan**

### CODE REVIEW

File: Pages/image.html [hunk 1]
```diff
@@ -379,12 +379,12 @@
...
 <div class="gallery">
-    <img src="../Content/images/photos_age/young%20age.jpg" alt="Merwan Sheriar Irani" width="300" height="200">
+   <img src="../Content/images/photos_age/young_age.jpg" alt="Merwan Sheriar Irani" width="130" height="253">
     <div class="desc">Baba at young age</div>
 </div>
 
 <div class="gallery">
-    <img src="../Content/images/photos_age/avatar.jpg" alt="Avatar Meher Baba" width="300" height="200">
+    <img src="../Content/images/photos_age/avatar.jpg" alt="Avatar Meher Baba" width="170" height="200">
     <div class="desc">Baba after Enlightened</div>
```
Comment (gsvarma) [inline Pages/image.html]: The caption of the third picture not visible on the webpage.

File: index.html [hunk 2]
```diff
@@ -69,6 +69,9 @@ <h3> Welcome to our website! </h3>
 <img src="Content/images/Avatar%20Meher%20Baba.jpg" alt="Baba image" style="float:right">
 
 <div id="footer">
+
+ <h4> <marquee behavior="alternate" BGCOLOR=White>Thank you for visiting meherbaba.in</marquee> <h4>
```
Comment (gsvarma) [inline index.html]: Looks good to me! nice addition. @vipasyan 👍

**Comment by vipasyan** [comment]
Resized Image

File: Pages/image.html (hunk 1 above)
Comment (vipasyan) [reply Pages/image.html]: worked on it, now everything seems fine.

File: Pages/content.html [hunk 3]
```diff
@@ -19,7 +19,7 @@
             position: fixed;
             left: 0px;
             bottom: 0px;
-            height: 70px;
+            height: 120px;
```
Comment (gsvarma) [inline Pages/content.html]: This is not specific to this PR #18.

File: Pages/quote.html [hunk 4]
```diff
@@ -19,7 +19,7 @@
             position: fixed;
             left: 0px;
             bottom: 0px;
-            height: 70px;
+            height: 120px;
```
Comment (gsvarma) [inline Pages/quote.html]: This is not specific to this PR #18.

**Pull request merged**